cylc/flow/scripts/trigger.py

```diff
@@ -17,11 +17,17 @@
 
 """cylc trigger [OPTIONS] ARGS
 
-Force tasks to run regardless of prerequisites.
+Manually trigger tasks regardless of prerequisites, even in a paused workflow.
```

Comment:
IMO avoid repeating the command verb in the description as that doesn't expand on what it does
```suggestion
Force task(s) to run regardless of prerequisites, even in a paused workflow.
```